NSS-Day-Cohort-49/giffygram-double-rainbows
# Fixed user profile page to display fav star

#### Changes Made

1. Modified file `Footer.js` to include add fav button to the posts
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout fix-user-profile-page
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does user goes to profile page they should see the fav button
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
